extensions/arc/src/test/stubs.ts

```diff
@@ -3,8 +3,54 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import * as azdata from 'azdata';
+import * as TypeMoq from 'typemoq';
 import * as vscode from 'vscode';
 
+export function createModelViewMock() {
```

Comment:
The basic template of these methods to create TypeMock mocks were share by @Charles-Gagnon. These were helpful in factoring the test code right. I am very thankful of his help here.